guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -207,19 +219,51 @@ export default {
 ```
 {% endcode %}
 
-- Override `sw-flow-sequence-action` to show custom action in action list
+Next, we override `sw-flow-sequence-action` component to show `CREATE_TAG` label in action list. For example, we override `getActionTitle` method to add icon, label for `CREATE_TAG` action.
 
 {% code title="<plugin root>/src/Resources/app/administration/src/extension/sw-flow-sequence-action/index.js" %}
 ```jsx
-import { ACTION } from '../../constant/example-plugin.constant';
+import { ACTION } from '../../constant/swag-example-plugin.constant';
+
+const { Component } = Shopware;
+
+Component.override('sw-flow-sequence-action', {
+    methods: {
+        getActionTitle(actionName) {
+            if (actionName === ACTION.CREATE_TAG) {
+                return {
+                    value: actionName,
+                    icon: 'default-badge-help',
+                    label: this.$tc('swag-example-plugin.titleCreateTag'),
+                }
+            }
+
+            return this.$super('getActionTitle', actionName);
+        },
+    },
+});
+```
+{% endcode %}
+
+Here is the result for the after the **Step 1**.
+
+![Choose a Flow Builder Action](../../../../../.gitbook/assets/flow-builder-trigger-action.png)
+
+### Step 2: Add configuration for action
+
+Before of that, we customise `modalName` to open configuration modal, `actionDescription` computed property and `getCreateTagDescription` to show action configuration description.
```

Comment:
"First, we customise `modalName` for the configuration modal, add an `actionDescription` computed property and create the `getCreateTagDescription` method to show action the configuration description."